commitlint.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+    rules: {
+        'type-enum': [
+            2,
+            'always',
+            [
```

Comment:
@fuhlig 
Let’s agree on a useful set of commit types. I’m not sure if we need all of these. Here is a set that I consider reasonable:

```js
[
    'build',
    'chore',
    'docs',
    'feat',
    'fix',
    'perf',
    'refactor',
    'release',
    'style',
    'test'
]
```

Reply:
Adjusted the types, please check

Reply:
Looks good, thanks!